rules/windows/persistence_remote_password_reset.toml

```diff
@@ -32,7 +32,7 @@ sequence by host.id with maxspan=5m
   [authentication where event.action == "logged-in" and
     /* event 4624 need to be logged */
     winlog.logon.type : "Network" and event.outcome == "success" and source.ip != null and
-    not source.ip in ("127.0.0.1", "::1")] by winlog.event_data.TargetLogonId
+    source.ip != "127.0.0.1" and source.ip != "::1"] by winlog.event_data.TargetLogonId
```

Comment:
and this change resolved the error on the stack?? Good to know that `in` requires keywords if so

Reply:
we should open an issue to validate that in our rules

Reply:
Yeah, it runs with no errors after the modification. I also tried to apply it this way on https://github.com/elastic/detection-rules/pull/1704 and noted the error on the description.